README.markdown

```diff
@@ -6548,6 +6548,8 @@ The optional `flags` argument specifies a user flags value associated with the e
 
 When it fails to allocate memory for the current key-value item, then `set` will try removing existing items in the storage according to the Least-Recently Used (LRU) algorithm. Note that, LRU takes priority over expiration time here. If up to tens of existing items have been removed and the storage left is still insufficient (either due to the total capacity limit specified by [lua_shared_dict](#lua_shared_dict) or memory segmentation), then the `err` return value will be `no memory` and `success` will be `false`.
 
+If the sizes of items in the dictionary are not approximate, it is eaisier to encounter `no memory` error because of memory fragmentation. It is recommended to use different dictionaries for different sizes of items. When you encounter `no memory` error, you can also remove more existing items by retrying more times to make the operation successful.
```

Comment:
What do you mean by "sizes are not approximate"? You mean they are not the same here right? Actually, if the sizes are all multiples or even powers of a certain value (like 2), it will also help too though there will still be a chance for fragmentation.

Also, this discussion also applies to other storing operations like `add()` and etc. We should add a separate section instead of repeating this in every storing operation's doc?

Also, better to use shorter lines here to help diffs in the future.

Reply:
Other storing operations already have pointed out that they are similar to the `set` method, so I think adding here is ok.